### ISSUE TITLE
accuracy

### ISSUE THREAD

**Issue by Radleye** [liruiw/Dec-SSL#2]
I run this command
`python src/decentralized_featarc_ssl_main.py --dataset=cifarssl --gpu=0 --iid=0  --dirichlet --dir_beta 0.02`
the accuracy is just 78.54%, the result in the paper is 84.63%，
Please tell me why , and do I need to adjust anything else?

**Comment by liruiw** [comment]
I might need to know more details about the training setup and hyperparameters to know exactly the reasons. I will also check it once I have more capacity.
